LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
+
+Copyright (c) 2017 Marvel
```

Comment:
Hello, thanks for contributing. Here we need 2017 Marvel Prototyping LTD

Reply:
updated! 👍